jobs/haproxy/templates/haproxy_wrapper.erb

```diff
@@ -101,7 +101,7 @@ trap reload_daemon USR2
 
 # Start up
 echo "$(date): Starting HAProxy"
-haproxy -f ${CONFIG} -W -p ${PID_FILE} <%= flags.join(' ') %>
+haproxy -f ${CONFIG} -W -p ${PID_FILE} <%= flags.join(' ') %> | cat -
```

Comment:
@mariash this only applies to when HAproxy is writing to stdout, right? can you wrap this pipe in a guard condition for that? 
Along the lines of line 91 where we add flags based on the `syslog_server` setting. You could introduce a `cat_pipe` variable that is `| cat -` if `run_in_foreground` and empty if not.